### PR TITLE
Improve the demo-warning.

### PR DESCRIPTION
**Pull request by GuGuss**
We need a link to Platform.sh and a very small explanation of it.

### CODE REVIEW

**Comment by xabbuh** [comment]
To me, the way it is shown is totally fine. :+1:

File: _theme/_templates/layout.html [hunk 1]
```diff
@@ -44,9 +44,9 @@
         <div id="sidebar" class="col-sm-3">
             <div id="sidebar-content">
                 <div id="demo-warning">
-                    <h4>This is a demo</h4>
-                    <p>This is a demo provided by platform.sh.<br>
-                    <a href="http://symfony.com/doc/current/{{ pagename }}">Visit on symfony.com</a>.</p>
+                    <h4>Pull request build</h4>
+                    <p>Each pull request of the Symfony Documentation is automatically deployed and hosted on <a href="https://platform.sh">Platform.sh</a>.<br>
+                    Visit the page on <a href="http://symfony.com/doc/current/{{ pagename }}">symfony.com</a>.</p>
```
Comment (xabbuh) [inline _theme/_templates/layout.html]: I think we should link to the HTTPS docs here.

**Comment by GuGuss** [comment]
thanks @xabbuh for your review.

**Comment by GuGuss** [comment]
:+1: 
@xabbuh Hi, any idea when that'll be merged? Thanks.

**Comment by xabbuh** [comment]
@weaverryan @WouterJ @javiereguiluz Any opinion on this one?

**Comment by javiereguiluz** [comment]
I like the reworded message and I think it's fair to add the link to platform.sh. My only concern is about the "Visit the page on symfony.com" message. I'm not sure if its intention is clear enough and by the way, it won't work when the page is new (which I know it's not a very frequent scenario).

Anyway, my concern can be also applied to the original message, so I'm :+1: about merging these changes.

**Comment by xabbuh** [comment]
I agree that we can handle this in a different pull request if we feel the need to do so.

**Comment by xabbuh** [comment]
Thank you Augustin.

**Comment by GuGuss** [comment]
@xabbuh @javiereguiluz you guys rock as reviewers! I wish there were more community projects like this one!

**Comment by WouterJ** [comment]
@javiereguiluz I added the "view this page on symfony.com" for 2 reasons: (a) People on the demo docs that just wanted to read the doc page can quickly navigate to the doc page and (b) I've used it a couple of times already to quickly create a visual diff (e.g. when someone fixes some syntax and I want to quicly check if it really did fix anything)

**Comment by javiereguiluz** [comment]
@WouterJ I agree and like this feature :)  My concern is about the text of the link. The original was *"Visit on symfony.com"* and the new one is *"Visit the page on symfony.com"*  I think that the text you included in your comment is much better: *"View this page on symfony.com"*

**Comment by weaverryan** [comment]
I've just updated the language

**Pull request closed without merging**